examples/src/amplist.html

```diff
@@ -34,11 +34,15 @@
       z-index: 2;
       padding: 16px;
     }
+    amp-list div[role="list"] {
+      display: grid;
+      grid-gap: 0.5em;
+    } 
   </style>
 </head>
 <body>
   <!-- basic -->
-  <amp-list width="auto" height="100" layout="fixed-height"
+  <amp-list id="testclass" width="auto" height="100" layout="fixed-height"
```

Comment:
why the id? also, why does it say "testclass" when it is an id,not a class ;)

Reply:
lol, for reasons that were real... thanks for the catch!